build-tools/automation/yaml-templates/run-installer.yaml

```diff
@@ -8,14 +8,16 @@ steps:
     artifactName: $(InstallerArtifactName)
     downloadPath: $(System.DefaultWorkingDirectory)
     patterns: xamarin.android*.pkg
-  condition: and(succeeded(), eq(variables['agent.os'], 'Darwin'))
+  # Disabled on .NET 6 release branches
+  condition: false
```

Comment:
I think we could keep this enabled to help make the macOS smoke test jobs more reliable, as they run a combination of classic and .NET tests.